CMakeLists.txt

```diff
@@ -95,6 +95,7 @@ endif()
 option(CYGWIN_USE_POSIX "Should the POSIX API be used for cygwin. Ignored if the system isn't cygwin." OFF)
 option(ENABLE_CXX11 "Should the c++11 parts (srt-live-transmit) be enabled" ON)
 option(ENABLE_APPS "Should the Support Applications be Built?" ON)
+option(ENABLE_TESTING "Should the Developer Test Applications be Built?" OFF)
 option(ENABLE_PROFILE "Should instrument the code for profiling. Ignored for non-GNU compiler." $ENV{HAI_BUILD_PROFILE})
```

Comment:
Do you expose this cmake option on purpose? Testing apps used to be for internal use.

Reply:
But the option exists and is handled. And sometimes I advice users that they should try with this one. Important thing is also that only the testing application is updated with appropriate changes to handle the socket groups.